code/rdb/datastripe.q

```diff
@@ -0,0 +1,3 @@
+\d .ds
+
+segmentid: "J"$@[{(.Q.opt .z.x)[x]};`segid;0n]
```

Comment:
add some descriptive comment after this plz.

Reply:
added comment